index.html

```diff
@@ -205,6 +205,166 @@ <h2 class="banner-title">
         </div>
       </div>
     </section>
+    <div class="categories">
+      <a class="tab-1" href="#">Bestsellers</a>
+      <a class="tab-2" href="#">Featured Products</a>
+    </div>
+
+    <div class="tab-container1">
+      <div class="bestsellers">
+        <div class="bestsellers-item">
+          <figure>
+            <img
+              src="/images/heart-hand-warmer.jpeg"
+              alt="heart hand warmer"
+              class="img-responsive2"
+            />
+          </figure>
+          <h4 class="item-name">Heart Hand Warmer</h4>
+          <h4 class="item-price">$5.99</h4>  
+          <div class="star-widget">
```

Comment:
this kind of solution is desired for star widget: https://codepen.io/hesguru/pen/BaybqXv